jujugui/static/gui/src/app/init/hotkeys.js

```diff
@@ -128,7 +114,7 @@ const activate = context => {
     const trigger = symbolic.join('-');
     const spec = keyBindings[trigger];
     if (spec) {
-      if (spec.condition && !spec.condition.call(context)) {
+      if (spec.condition && !spec.condition.call()) {
```

Comment:
Why are you explicitly calling the condition with an `undefined` context?

Reply:
haha, oops.